emily/handler/src/api/handlers/deposit.rs

```diff
@@ -213,7 +213,24 @@ pub async fn create_deposit(
         context: EmilyContext,
         body: CreateDepositRequestBody,
     ) -> Result<impl warp::reply::Reply, Error> {
-        // Set variables.
+        // Reject dups.
+        let (entries, _next_token) =
+            accessors::get_deposit_entries(&context, &Status::Accepted, None, None).await?;
+        let deposits: Vec<DepositInfo> = entries.into_iter().map(|entry| entry.into()).collect();
+        for deposit in deposits {
+            if deposit.bitcoin_txid == body.bitcoin_txid
+                && deposit.bitcoin_tx_output_index == body.bitcoin_tx_output_index
+                && deposit.status == Status::Accepted
+            {
+                let responce = json(&serde_json::json!({
+                    "error": "Conflict",
+                    "message": "This deposit already exists and accepted."
+                }));
```

Comment:
Looking around we return `Error` directly (`Error::Conflict` here) and let the `impl Error` handle the required serialization.